app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+
+import { Button, Pagination, Alert, Toastr } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import MenuBar from "./MenuBar";
+import Table from "./Table";
+
+const Contacts = () => {
+  const [searchTerm, setSearchTerm] = useState("");
+  // const [showNewContactPane, setShowNewContactPane] = useState(false);
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+
+  const handleDeleteAlert = () => {
+    setShowDeleteAlert(showDeleteAlert => !showDeleteAlert);
+  };
+
+  const handleSubmit = () => {
+    setShowDeleteAlert(showDeleteAlert => !showDeleteAlert);
+    Toastr.success("Contact deleted successfully.");
+  };
+
+  return (
+    <>
+      <MenuBar />
+      <Container>
+        <Header
+          menuBarToggle
+          title="All Contacts"
+          actionBlock={
+            <Button
+              icon="ri-add-line"
+              label="Add Contacts"
+              // onClick={() => setShowNewContactPane(true)}
```

Comment:
This should've been caught in your self-review. Why push comments into PR? What benefit does it add? Please remove this. Commenting a code means, it's unnecessary.

Read: https://handbook.neetokb.com/articles/git-and-github-essentials#self-review-process